draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -170,8 +170,8 @@
 <t>
    Each HTTP message is either a request or a
    response. A server listens on a connection for a request, parses each
-   message received, interprets the message semantics in relation to the
-   identified request target, and responds to that request with one or more
+   message received, interprets the message semantics in relation to
+   its target resource, and responds to that request with one or more
```

Comment:
This would be better as "in relation to the identified target resource" (both to remove ambiguity and to emphasize that the message provides that identifier).

Reply:
Done in local source